tests-invoke

```diff
@@ -19,257 +19,82 @@
 
 import argparse
 import os
-import socket
 import subprocess
 import sys
 import time
 
-sys.dont_write_bytecode = True
-
 from task import github
-from task import sink
 from machine.machine_core.directories import BASE_DIR
 
-
-HOSTNAME = socket.gethostname().split(".")[0]
-DEVNULL = open("/dev/null", "r+")
+sys.dont_write_bytecode = True
 
 
 def main():
     parser = argparse.ArgumentParser(description='Run integration tests')
-    parser.add_argument('--publish', dest='publish', default=os.environ.get("TEST_PUBLISH", ""),
-                        action='store', help='Publish results centrally to a sink')
     parser.add_argument('-v', '--verbose', action='store_true', help='Verbose output')
-    parser.add_argument('--pull-number', help="The number of the pull request to test")
-    parser.add_argument('context', help="The context or type of integration tests to run")
+    parser.add_argument('--revision', help="Revision of the PR head", required=True)
+    parser.add_argument('--base-project', help="Project in which the tested PR is opened",
+                        required=True)
+    parser.add_argument('--pull-number', help="The number of the pull request to test",
+                        required=True)
+    parser.add_argument('--context', help="Test context", required=True)
```

Comment:
I have a gut feeling that this should be dropped, and setting `TEST_{OS,SCENARIO}` should be put into the command line by tests-scan directly. This avoids propagating the context syntax into too many different scripts.

But this is fine for a follow-up, of course.